app/controllers/application_controller.rb

```diff
@@ -48,7 +48,7 @@ def log_invalid_token_attempt
   end
 
   def state_objects
-    @state_objects ||= SortCollectionOrdinally.call(State.all)
+    @state_objects ||= SortCollectionOrdinally.call(collection: State.all, column_name: e.send(:column_name))
```

Comment:
We don't need the second parameter here, since states have names in our schema.

Reply:
Oh, ok - so when I talk about using `send`, I mean in the actual service object code, not here.